Sources/InfomaniakCoreCommonUI/MatomoUtils.swift

```diff
@@ -54,6 +72,26 @@ public final class MatomoUtils {
             value: Float(number)
         )
     }
+
+    private func logTrack(category: EventCategory, action: UserAction, name: String, value: Any?) {
```

Comment:
```suggestion
    private func logTrack(category: EventCategory, action: UserAction, name: String, value: Float?) {
```
It will always be a Float